src/isotools/_gene_plots.py

```diff
@@ -445,6 +459,11 @@ def gene_track(self, ax=None, title=None, reference=True, select_transcripts=Non
             blocked.append(tr_end)
         else:
             blocked[i] = tr_end
+        
+        # use SQANTI color palette if colorbySqanti is True
+        if colorbySqanti:
+            color = sqanti_palette[tr['annotation'][0]]['color']
```

Comment:
Execution failed here for me with `KeyError: 'annotation'`. I would add a check if the transcript is already annotated and if not, add an info log and stick to the normal color scheme.
The comment is also rather redundant.